pkg/sdn/plugin/vnids_node.go

```diff
@@ -26,10 +24,7 @@ type nodeVNIDMap struct {
 }
 
 func newNodeVNIDMap() *nodeVNIDMap {
-	return &nodeVNIDMap{
-		ids:        make(map[string]uint32),
-		namespaces: make(map[uint32]sets.String),
-	}
+	return &nodeVNIDMap{}
```

Comment:
No, it's part of a set of changes to get rid of plugin.go:getVNID(); the fields get initialized by VnidStartNode() now, which only gets called for multitenant, so if GetVNID() sees that they haven't been initialized later, it can just return 0 for the VNID, rather than plugin.go having to make that assumption itself.